BedrockServer.cpp

```diff
@@ -788,6 +788,22 @@ void BedrockServer::runCommand(unique_ptr<BedrockCommand>&& _command, bool isBlo
         }
     }
 
+    // If we're on a different version from leader AND this is not a command that was escalatedImmediatly, 
+    // we'll  return a 542 error so that the client knows it needs to call another server from the cluster. 
+    // We're doing this to prevent overloading leader with requests from all servers instead of spreading 
+    // them equaly between available servers.
```

Comment:
This comment is entirely wrong, correct?

Reply:
Yes, thanks for noticing.